### PR TITLE
Formatter package 

### PR DESCRIPTION
**Pull request by arjunu**
- moved formatter utils to shared
- replaced `<Money/>` with `<Eth/>`, `<Eur/>`, `<EurToken/>`, `<Neu/>`
- used class to apply `ETheme.GREEN`, other themes are not used in the app anywhere

Outliers:
### Eth
- [ ] packages/web/app/components/modals/tx-sender/investment-flow/Investment.tsx#L367 
- [ ] packages/web/app/components/modals/tx-sender/investment-flow/utils.tsx#L162

### Eur
- [ ] packages/web/app/components/eto/overview/EtoOverviewStatus/CampaigningWidget/CampaigningActivatedInvestorApprovedWidgetLayout.tsx#L72

### CODE REVIEW

File: packages/design-system/src/components/formatters/EurToken.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  TDataTestId,
+} from "@neufund/shared";
+import * as React from "react";
+
+import { Units } from "./atoms/Units";
+import { IValueProps, Value } from "./atoms/Value";
+import { ICommonMoneyProps } from "./types";
+import { formatCurrency } from "./utils";
+
+export const EurToken: React.FunctionComponent<IValueProps & ICommonMoneyProps & TDataTestId> = ({
```
Comment (Moejoe90) [inline packages/design-system/src/components/formatters/EurToken.tsx]: same as mentioned before regarding the interfaces

File: packages/design-system/src/components/formatters/Neu.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  TDataTestId,
+} from "@neufund/shared";
+import * as React from "react";
+
+import { Units } from "./atoms/Units";
+import { IValueProps, Value } from "./atoms/Value";
+import { ICommonMoneyProps } from "./types";
+import { formatCurrency } from "./utils";
+
+export const Neu: React.FunctionComponent<IValueProps & ICommonMoneyProps & TDataTestId> = ({
```
Comment (Moejoe90) [inline packages/design-system/src/components/formatters/Neu.tsx]: As mentioned before regarding the interfaces

File: packages/web/app/components/dashboard/my-portfolio/my-neu-widget/MyNeuWidgetAvailalblePayout.tsx [hunk 3]
```diff
@@ -28,16 +22,7 @@ export const MyNeuWidgetAvailablePayout: React.FunctionComponent<TAvailablePayou
...
-            <Money
-              value={tokensDisbursalEurEquiv}
-              inputFormat={ENumberInputFormat.ULPS}
-              valueType={ECurrency.EUR}
-              outputFormat={ENumberOutputFormat.FULL}
-              theme={ETheme.GREEN}
-              className={styles.payoutAmount}
-            />
-          ),
+          amount: <Eur className={styles.payoutAmount} value={tokensDisbursalEurEquiv} />,
```
Comment (Moejoe90) [inline packages/web/app/components/dashboard/my-portfolio/my-neu-widget/MyNeuWidgetAvailalblePayout.tsx]: Looks so much better :+1:

File: packages/design-system/src/components/formatters/Eur.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import { IValueProps, Value } from "./atoms/Value";
+import { ICommonMoneyProps } from "./types";
+import { formatCurrency } from "./utils";
+
+export const Eur: React.FunctionComponent<IValueProps & ICommonMoneyProps & TDataTestId> = ({
+  className,
+  value,
+  defaultValue,
+  roundingMode = ERoundingMode.DOWN,
+  inputFormat = ENumberInputFormat.ULPS,
```
Comment (Moejoe90) [inline packages/design-system/src/components/formatters/Eur.tsx]: I'm if the default value is really `ULPS` 
It should be `float` anything that is ULPS should be `EUR_TOKEN` IMO

File: packages/design-system/src/components/formatters/Eth.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  roundingMode = ERoundingMode.DOWN,
+  inputFormat = ENumberInputFormat.ULPS,
+  outputFormat = ENumberOutputFormat.ONLY_NONZERO_DECIMALS,
+  "data-test-id": dataTestId,
+}) => {
+  const formattedValue =
+    value &&
+    formatCurrency({
+      value,
+      valueType: ECurrency.EUR,
```
[the remark this answers is not in the document]
Comment (arjunu) [reply packages/design-system/src/components/formatters/Eth.tsx]: this was a copying mistake, fixed

[the remark this answers is not in the document]
Comment (arjunu) [reply packages/design-system/src/components/formatters/Eth.tsx]: fixed

**Comment by arjunu** [comment]
Closed for now

**Pull request closed without merging**